src-tauri/src/lib.rs

```diff
@@ -299,7 +299,7 @@ pub fn run() {
         .setup(|app| {
             let window = app.get_webview_window("main").unwrap();
 
-            DB::init().map_err(|e| e.to_string())?;
+            DB::init(DbPassword::Keychain).map_err(|e| e.to_string())?;
```

Comment:
Why pass db::DbPassword::Keychain into db::DB::init instead of just calling DbPassword::Keychain from inside the fn? Is there a future scenario in mind?